packages/core/content-manager/admin/src/preview/components/PreviewSidePanel.tsx

```diff
@@ -58,6 +58,8 @@ const PreviewSidePanel: PanelComponent = ({ model, documentId, document }) => {
     trackUsage('willNavigate', { from: pathname, to: destinationPathname });
   };
 
+  const isDisabled = isModified;
```

Comment:
why do you need to do create a new variable? Couldn't you just use isModified wherever you need things disabled?

Reply:
I did that because I check in [3 different places](https://github.com/strapi/strapi/pull/22324/files#diff-a3f6f0d2c32a2510e10592dc1904c0d9b8770cb2855aff5efd2375e233bac15fR81-R83) if the button is disabled, so if in the future there are more reasons to disable then we'd need to change the check everywhere.

We don't have that problem for now though, so I'll update to simplify 👍

Reply:
Gotcha 👍 , clever. But I agree with removing it for now or leaving a comment to explain your intention (otherwise next time I'm in that file 🧹 )